File: function/newArray.cpp
//Link: https://codeforces.com/group/MWSDmqGsZm/contest/223205/problem/L

#include <iostream>
#include <algorithm>
#include<cmath>
#include <vector>
#include <iomanip>
using namespace std;

void concatinationArray(vector<int>&a, vector<int>&b)
{
	int n = a.size() + b.size();
	vector<int>c;
	c.insert(c.end(), b.begin(), b.end());
	c.insert(c.end(), a.begin(), a.end());

	for (int i = 0; i < c.size(); i++)
	{
		if (i == c.size() - 1)
			cout << c[i];
		else
			cout << c[i] << " ";
	}
}

int main()
{
	int n;
	cin >> n;
	vector<int>a(n);
	vector<int>b(n);
	for (int i = 0; i < n; i++)
	{
		cin >> a[i];
	}

	for (int i = 0; i < n; i++)
	{
		cin >> b[i];
	}

	concatinationArray(a, b);

}

File: src/NewArray.java
//Link: https://codeforces.com/group/MWSDmqGsZm/contest/223205/problem/L

import java.util.ArrayList;
import java.util.Collection;
import java.util.Collections;
import java.util.Scanner;

public class NewArray
{
    public static void main(String[]args)
    {
        Scanner cs =new Scanner(System.in);
        int n=cs.nextInt();
        ArrayList<Integer>a=new ArrayList<>(n);
        ArrayList<Integer>b=new ArrayList<>(n);
        for(int i=0;i<n;i++)
        {
            a.add(cs.nextInt());
        }

        for(int i=0;i<n;i++)
        {
            b.add(cs.nextInt());
        }

        concatinationArray(a, b);
    }
    private static void concatinationArray(ArrayList<Integer>a,ArrayList<Integer> b)
    {
        int n=a.size()+b.size();
        ArrayList<Integer>c=new ArrayList<>(n);
        c.addAll(b);
        c.addAll(a);

        for(int i=0;i<n;i++)
        {
            if(i==n-1)
                System.out.print(c.get(i));
            else
                System.out.print(c.get(i)+" ");
        }
    }
}
